### DISCUSSION TITLE
/siege

### DISCUSSION THREAD

**Discussion by crosskill45** [GriefPrevention/GriefPrevention#914] [Q&A] [answered]
how do enable /siege command because i cant find it in the config

**Comment by RoboMWM** [comment, marked as answer]
you add it to the section that holds the list of worlds that siege is enabled in. Pretty sure the documentation says something about it too. It's probably blank in your config, so you'll need to change `[]` to `[your_world_here]` or if you want more worlds, `[your_world_here, another_world_here]`
